webapp/src/components/Dialogs/LogsDialog.jsx

```diff
@@ -0,0 +1,91 @@
+import React, { useEffect, useState } from 'react';
+
+import PropTypes from 'prop-types';
+
+import Button from '@material-ui/core/Button';
+import Checkbox from '@mui/material/Checkbox';
```

Comment:
I think we are mixing the mui and material-ui libraries here.
The mui library is newer than the old material-ui package. Here in the webapp the old material-ui is still used so it would be good to not mix it with the mui library. At some point we could switch the entire webapp to mui.

Reply:
I've removed @mui/material from the dependencies and used material-ui instead in the new commit. Thanks for pointing this out.